include/tvm/relay/attrs/vision.h

```diff
@@ -40,6 +40,22 @@ struct MultiBoxPriorAttrs : public tvm::AttrsNode<MultiBoxPriorAttrs> {
   }
 };
 
+/*! \brief Attributes used in non_maximum_suppression operators */
+struct NMSAttrs : public tvm::AttrsNode<NMSAttrs>{
+  double nms_threshold;
```

Comment:
do we need nms prefix here? as it is already an nms attribute

Reply:
suggest using overlap_threshold, for separation with score_threshold, etc.